tests/unit/converters/test_base_converter.py

```diff
@@ -168,3 +168,13 @@ def test_load_data(converter: DummyConverter):
     converter_2 = DummyConverter(source=source)
     converter_2._load_data(data=None)
     source.load.assert_called_once()
+
+
+def test_base_converter_log_level():
+    import logging
```

Comment:
please move this import to the top. no harm in importing it for the other tests

Reply:
Suggestion included in the newest commit. Conversation resolved.